### PR TITLE
Issue 40

### PR DESCRIPTION
**Pull request by timronan**
Add patch to deal with the issue 40 enhancement. 
Please note that volume.build was moved from XmlToSeedDocumentConverter to XmlToSeedFileConverter. XmlToSeedFileConverter calls XmlToSeedDocumentConverter and  XmlToSeedFileConverter  accepts input arguments. It seems as though there is no reason to build the volume until after the volume can be manipulated by user defined arguments.

### CODE REVIEW

**Comment by timronan** [comment]
This pull request is designed to address issues 37 and 40.

**Comment by timronan** [comment]
The <StationXML prologue check has been extended to search through the first 255 characters and be case sensitive by commit 1ed1aa4574d30533841c74dcc69f19225d8310d3. 

Is there a good tag for automatic dateless detection? Maybe the first 10 characters must be `000001V 01`? Is this robust enough to work on any dataless file?

**Comment by chad-iris** [comment]
> Is there a good tag for automatic dateless detection? Maybe the first 10 characters must be 000001V 01? Is this robust enough to work on any dataless file?

That's probably the best one can do.  If there is a test for it I suggest just a warning if it doesn't pass and let it default to SEED.

P.S. the "<StationXML" is not a prologue (sic), it is simply the root element of the document.  A prolog is the stuff before the root element as described here: http://tizag.com/xmlTutorial/xmlprolog.php.  Note the size of the prolog in the "XML & DTD Declaration Together" section, this is why 100 characters is not enough.

**Comment by timronan** [comment]
We are currently defaulting to SEED, which seems to be the more robust approach than trying to auto-detect dataless files. 

> suggest just a warning if it doesn't pass
Should I add this warning trigger? I am a little bit worried about spamming stdout but it could be a good sequence number verification. 

Thanks for the prolog explanation. That is an important xml component to recognize and describe accurately.

File: src/main/java/edu/iris/dmc/station/Application.java [hunk 1]
```diff
@@ -115,27 +126,17 @@ private void convert(File source, File target, Map<String, String> map)
...
-	              char inputChar = (char) data;
-	              extentionString +=inputChar;
-	              data = r.read();
-	              i = i+1;
-	           } 			
-			if (extentionString.contains(xmlstring)) {
+			String extension = null;
+
+			if (isStationXml(source)) {
+				logger.info("File type is StationXml");
```
Comment (chad-iris) [inline src/main/java/edu/iris/dmc/station/Application.java]: The name of the format is "StationXML", no code-speak in user-visible messages please.

Comment (timronan) [reply src/main/java/edu/iris/dmc/station/Application.java]: Do we need to add a message to the log the detected file type? This seems like extraneous information.

File: src/main/java/edu/iris/dmc/station/Application.java [hunk 2]
```diff
@@ -58,11 +66,14 @@ public void run(String... args) {
...
-				 * h.setLevel(Level.INFO); } logger.log(Level.FINEST, "SEED >< XML CONVERTER");
-				 */
+
+				Logger rootLogger = LogManager.getLogManager().getLogger("");
+				rootLogger.setLevel(Level.INFO);
+				for (Handler h : rootLogger.getHandlers()) {
+					h.setLevel(Level.INFO);
+				}
+				logger.log(Level.FINEST, "SEED >< XML CONVERTER");
+
```
Comment (timronan) [inline src/main/java/edu/iris/dmc/station/Application.java]: Can we switch the verbosity level to ERROR or WARN? INFO seems to be too verbose the the loader prints out :

> Mar 17, 2020 3:49:08 PM edu.iris.dmc.station.Application convert
> INFO: Assume file type is SEED
 
every time it is run.

File: src/main/java/edu/iris/dmc/station/Application.java (hunk 1 above)
Comment (timronan) [reply src/main/java/edu/iris/dmc/station/Application.java]: These comments would probably be alright if we change the logger level to DEBUG and change our default logger level to error or warnings.

File: src/main/java/edu/iris/dmc/station/Application.java (hunk 2 above)
Comment (timronan) [reply src/main/java/edu/iris/dmc/station/Application.java]: This conversion may be resolved by commit 49c41ddd5f9036643484817a78adc789f5aada79. Please review and comment.

**Comment by timronan** [comment]
Unit test were added and branch is ready to merge with master. Please verify.

**Pull request merged**